src/Monolog/Handler.php

```diff
@@ -41,6 +41,10 @@ public function __construct(HubInterface $hub, $level = Logger::DEBUG, bool $bub
 
     /**
      * {@inheritdoc}
+     *
+     * @psalm-suppress MoreSpecificImplementedParamType
+     *
+     * @param array{level:int, level_name:string, message:string, channel:string, context:mixed} $record
```

Comment:
This should be fixed upstream. Anyway, can you please format the shape definition on multiline?

Reply:
Done.